.circleci/config.yml

```diff
@@ -1235,50 +1267,51 @@ jobs:
       - magma_slack_notify
 
 workflows:
-  version: 2
-
-  cloud:
-    jobs:
-      - insync-checkin
-      - cloud_lint
-      - cloud-test
-      - orc8r-build:
-          requires:
-            - insync-checkin
-            - cloud_lint
-            - cloud-test
-
-  lib_gateway:
-    jobs:
-      - orc8r-gateway-test
+  version: 2.1
+
+  # cloud:
+  #   jobs:
+  #     - insync-checkin
+  #     - cloud_lint
+  #     - cloud-test
+  #     - orc8r-build:
+  #         requires:
+  #           - insync-checkin
+  #           - cloud_lint
+  #           - cloud-test
+  #
+  # lib_gateway:
+  #   jobs:
+  #     - orc8r-gateway-test
 
   agw:
     jobs:
       - lte-test
       - session_manager_test
       - mme_test
       - mme-clang-tidy:
-          <<: *only_master
+           <<: *only_master
       - mme-clang-warnings:
-          <<: *only_master
+           <<: *only_master
       - lte-integ-test:
-          <<: *master_and_develop
+           <<: *master_and_develop
       - lte-agw-deploy:
-          <<: *only_master
-          requires:
-            - lte-test
-            - mme_test
-            - lte-integ-test
-      - c-cpp-codecov
-
+           <<: *only_master
+           requires:
+             - lte-test
+             - mme_test
+             - lte-integ-test
+      - lte-agw-aws:
```

Comment:
Let's make lte-agw-deploy required

Reply:
lte-agw-deploy is required is it not?

Reply:
done

Reply:
- lte-agw-aws:
           <<: *only_master
           requires:
              - lte-agw-deploy